datajunction-server/datajunction_server/construction/build.py

```diff
@@ -78,6 +77,9 @@ def _join_path(
             if col.dimension and col.dimension.type == NodeType.DIMENSION:
                 dimensions_to_columns[col.dimension.current].append(col)
 
+        # for link in current_node.dimension_links:
```

Comment:
forgot to uncomment or not needed?

Reply:
Oh yeah, this is no longer needed. Will remove.